### PR TITLE
Generate default configuration file

### LINKED ISSUE #9: Generate a default configuration if no config exists
Currently, if no configuration is made, the program will crash. When attempting to read the configuration, if the `raudio_config.json` file does not exist, the program should create one for them. The interaction for this would look like this:

```
$ python3 raudio_bot
Currently, there is no configuration specified for this bot. Would you like to create one? (Y/n)
What is the URL of the server you will be streaming from? <wait for input>
```

The only required field for the bot to run is the `server_url`, so you must prompt them to enter it from the command line. After this interaction, a `raudio_config.json` should be created with default values for every field. The implementation should be scalable (i.e., it will adapt if more configuration options are set in the future.)

### PR DESCRIPTION
**Pull request by cmille47**
Attempted to resolve issue #9. Prompts user is a configuration file is not found and will generate one based on user input.

### CODE REVIEW

**Comment by simonrodrig** [comment]
Hey @cmille47, thank you for submitting a pull request! I will be reviewing it shortly.

File: raudio_bot/__main__.py [hunk 1]
```diff
@@ -10,6 +10,36 @@
...
+        return True
+    else:
+        x = input("Currently, there is no configuration specified for this bot. Would you like to create one? (y/n): ")
+
+        if (x == 'y'):
+            # get url and create data dictionary containing default specifications
+            url = input("Enter URL of server you will be streaming from: ")
+
+            # ADD ANY OTHER DEFAULT CONFIGS HERE
+            data = {'stream_url': url, 'authenticated': [ ]}
```
Comment (simonrodrig) [inline raudio_bot/__main__.py]: Ideally we will want to interface with our `RaudioConfig` class instead, which would save us from needing to specify the config in two places. You can declare it like any other class.

```suggestion
            data = RaudioConfig(stream_url=url, authenticated=[])
```

File: raudio_bot/__main__.py [hunk 2]
```diff
@@ -10,6 +10,36 @@
...
+
+            # create default json file in upper directory
+            with open('raudio_config.json','w') as jsonfile:
+                json.dump(data, jsonfile)
+
+            # call recursively on newly created file
+            raudio_config_from_json('raudio_config.json')
```
Comment (simonrodrig) [inline raudio_bot/__main__.py]: Calling this function again doesn't seem necessary.

```suggestion
```

**Review by simonrodrig — CHANGES_REQUESTED** [review]

File: raudio_bot/__main__.py (hunk 2 above)
Comment (cmille47) [reply raudio_bot/__main__.py]: Ya that makes sense. Would I just commit your suggestion to clear this up?

Comment (simonrodrig) [reply raudio_bot/__main__.py]: Yup!

File: raudio_bot/__main__.py [hunk 3]
```diff
@@ -10,6 +10,35 @@
...
+        return True
+    else:
+        x = input("Currently, there is no configuration specified for this bot. Would you like to create one? (y/n): ")
+
+        if (x == 'y'):
+            # get url and create data dictionary containing default specifications
+            url = input("Enter URL of server you will be streaming from: ")
+
+            # ADD ANY OTHER DEFAULT CONFIGS HERE
+            data = RaudioConfig(stream_url=url, authenticated=[])
```
Comment (simonrodrig) [inline raudio_bot/__main__.py:27]: RaudioConfig isn't imported. You'll also need to add the import at the top

**Review by simonrodrig — APPROVED** [review]

**Pull request merged**